ts/features/bonus/cgn/saga/orchestration/activation/activationSaga.ts

```diff
@@ -0,0 +1,32 @@
+import { SagaIterator } from "redux-saga";
+import { call, put, race, take } from "redux-saga/effects";
+import { NavigationActions } from "react-navigation";
+import { navigationHistoryPop } from "../../../../../../store/actions/navigationHistory";
+import {
+  cgnActivationCancel,
+  cgnActivationComplete,
+  cgnRequestActivation
+} from "../../../store/actions/activation";
+import { navigateToCgnActivationInformationTos } from "../../../navigation/actions";
+
+export function* cgnStartActivationWorker() {
+  yield put(navigateToCgnActivationInformationTos());
+  yield put(navigationHistoryPop(1));
```

Comment:
Keep in mind that this saga could be called from different part of the app and popping 1 screen from the stack could be wrong in certain cases (es: from a CTA in the message)

This is not an error but, since you have more context, considering the usage of `saga workunit`

Reply:
I just didn't actually realized how to integrate the workunit in this case, but now I did! 😄  you can find in 3d34b5a